File: mario.c
#include <cs50.h>
#include <stdio.h>
#include <math.h> 


int main(void)
{
    int height;
    int aatte;
    do 
/// to ask for height
    {
        height = get_int("height: \n");
        aatte = height;
    }
    while (height <= 0 || 8 < aatte);
    
    /// printing the height
    int spaces;
    for (int i = 0; i < height; i++)
    {
        /// the amount of spaces left side
        for (spaces = 1; spaces < height - i; spaces++)
        {
            printf(" ");
        }
        /// # left side
        for (int j = -1; j < i; j++)
        {
            printf("#");
        }
        /// space in middle
        for (int k = 0; k < 2; k++)
        {
            printf(" ");
        }
        /// # on right side
        for (int p = -1; p < i; p++) 
        {
            printf("#");
        }
        printf("\n");
    }
}
 

File: Python/pset6/mario.py
from cs50 import *

height = 0

while height > 8 or height < 1:
    # get height from user
    height = get_int("Height:")


else:
    for i in range(height):
        for spaces in range(1, height - i):
            print(" ", end="")
    
        # left side
        for j in range(-1, i):
            print("#", end="")
            
        for k in range(2):
            print(" ", end="")
    
        # right side
        for p in range(-1, i):
            print("#", end="")
        print("\n", end="")
    
